File: Python/human.py
class Human: # Deklarasi class Human (parent class)
    __identity = 0 # Untuk menyimpan NIK (Identity Number) mahasiswa, atribut berjenis private
    __name = "" # Untuk menyimpan nama mahasiswa, atribut berjenis private
    __gender = '' # Untuk menyimpan jenis kelamin mahasiswa, atribut berjenis private

    def __init__(self, identity = 0, name = "", gender = ''): # Membuat konstruktor
        self.__identity = identity # Menginisialisasi atribut __identity
        self.__name = name # Menginisialisasi atribut __name
        self.__gender = gender # Menginisialisasi atribut __gender

    def setIdentity(self, identity): # Setter untuk atribut __identity
        self.__identity = identity # Menginisialisasi atribut __identity

    def setName(self, name): # Setter untuk atribut __name
        self.__name = name # Menginisialisasi atribut __name

    def setGender(self, gender): # Setter untuk atribut __gender
        self.__gender = gender # Menginisialisasi atribut __gender

    def getIdentity(self): # Getter untuk atribut __identity
        return self.__identity # Mengembalikan nilai atribut __identity

    def getName(self): # Getter untuk atribut __name
        return self.__name # Mengembalikan nilai atribut __name

    def getGender(self): # Getter untuk atribut __gender
        return self.__gender # Mengembalikan nilai atribut __gender

File: C++/Human.cpp
// Import library yang dibutuhkan
#include <bits/stdc++.h>

using namespace std; // Menggunakan standard namespace

class Human // Deklarasi class Human (parent class)
{
private:                // Membuat atribut berjenis private
    long long identity; // Untuk menyimpan NIK mahasiswa
    string name;        // Untuk menyimpan nama mahasiswa
    char gender;        // Untuk menyimpan jenis kelamin mahasiswa

public:
    /* Konstruktor */
    Human() // Konstruktor tanpa isian
    {
        this->identity = 0LL;
        this->name = "";
        this->gender = '-';
    }

    Human(long long identity, string name, char gender) // Konstruktor dengan isian dari parameter
    {
        this->identity = identity;
        this->name = name;
        this->gender = gender;
    }

    /* Setter dan Getter untuk setiap atribut di dalam class Human*/
    void setIdentity(long long identity)
    {
        this->identity = identity;
    }

    void setName(string name)
    {
        this->name = name;
    }

    void setGender(char gender)
    {
        this->gender = gender;
    }

    long long getIdentity()
    {
        return this->identity;
    }

    string getName()
    {
        return this->name;
    }

    char getGender()
    {
        return this->gender;
    }

    /* Destruktor */
    ~Human()
    {
    }
};